homeassistant/components/folder/sensor.py

```diff
@@ -96,6 +98,7 @@ def device_state_attributes(self):
             "filter": self._filter_term,
             "number_of_files": self._number_of_files,
             "bytes": self._size,
+            "fileList": self._file_list,
```

Comment:
Should be snake-case.

```suggestion
            "file_list": self._file_list,
```